docs/index.md

```diff
@@ -1,7 +1,10 @@
 # PMM Dump Documentation
-
+    
 The `pmm_dump` client utility performs a logical backup of the performance metrics, collected by the PMM Server and imports them into a different PMM Server instance.
 
 PMM Dump allows you to share monitoring data, collected by your PMM server, with Percona Support team securely.
 
 
+!!! hint alert alert-success "Important"
+    Starting with PMM 2.41, the standalone client utility has been seamlessly integrated into PMM and is easily accessible from the main menu at > Help > PMM Dump. 
```

Comment:
There are plenty of other reasons to use command-line version of PMM Dump. For example, current integration does not support filtering options. So we need to have proper user for the integration, not confuse users that integration replaces standalone version.